benches/signatures.rs

```diff
@@ -30,26 +29,23 @@ fn native_keypair(c: &mut Criterion) {
 struct SigningData {
     k: RistrettoSecretKey,
     p: RistrettoPublicKey,
-    r: RistrettoSecretKey,
-    m: RistrettoSecretKey,
+    m: [u8; 32],
```

Comment:
From the current discussions around sensitive data being 'hidden' and/or zeroize. Shouldn't we heap allocate the data, using `Box` and wrap it around something like the `Hidden` type?

Reply:
Does not matter for tests or benchmarks.